src/Utility/AnaSol.h

```diff
@@ -125,7 +126,7 @@ class AnaSol
       return scalSecSol.size() <= idx ? nullptr : scalSecSol[idx];
 
     return scalSecSol.size() <= nPatches*idx+patch ?
-                                nullptr : scalSecSol[nPatches*idx+patch];
+                                nullptr : scalSecSol[nPatches*idx + patch];
```

Comment:
The body of getScalarSol and getScalarSecSol should now be placed in the C-file instead.